index.html

```diff
@@ -25,59 +28,60 @@
       </nav>
     </header>
     <main>
-      <h1>
-        WITH BINABOX EXPLORE NFT COLLECTION
+      <div class="binabox">
+      <h1 class="binabox">
+        WITH <span>BINABOX  </span> EXPLORE NFT COLLECTION
       </h1>
       <p>
         We are the best way to check the rarity of NFT
         collection
       </p>
-      <div>2240+ <small>Total Items</small></div>
-      <div>1000+ <small>Profiles Whitelisted</small></div>
-      <button>Connect Wallet</button>
-      <button>Whitelist Now</button>
-      <section id="Catalog">
-        <h2>Catalog</h2>
-        <button>See all</button>
-        <button>&lt;</button>
-        <button>&gt;</button>
-
+      <div class="binabox__details">2240+ <small>Total Items</small></div>
+      <div class="binabox__details">1000+ <small>Profiles Whitelisted</small></div>
+      <button class="binabox__button">Connect Wallet</button>
+      <button class="binabox__button">Whitelist Now</button>
+      </div>
+      <section id="catalog">
+        <h2 class="catalog" >Catalog</h2>
+        <button class="catalog-button">&lt;</button>
+        <button class="catalog-button">&gt;</button>
+        <button class="catalog-button__see">See all</button>
```

Comment:
two classes here: "catalog__button"  and "catalog__button--see"
first one is for block__element, second is basic class with modifier